File: api/Desafio.Application/Models/Usuarios/UsuarioModel.cs
using Desafio.Application.Models.AtivosUsuario;
using Desafio.Application.Models.ContasCorrentes;
using System;
using System.Collections.Generic;

namespace Desafio.Application.Models.Usuarios
{
    public class UsuarioModel
    {
        public Guid Id { get; set; }
        public string Nome { get; set; }

        public ContaCorrenteModel ContaCorrente { get; set; }
        public List<AtivoUsuarioModel> AtivosUsuario { get; set; }
    }
}

File: clientApp/src/app/models/Usuario.model.ts
import { ContaCorrente } from "./ContaCorrente.model";

export class Usuario {
  public id: string = '';
  public nome: string = '';
  
  public ContaCorrente: ContaCorrente = new ContaCorrente();
  public AtivosUsuario: Array<any> = new Array<any>();
}
